File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/CDAutonBaseTest.java
package org.firstinspires.ftc.teamcode;

import com.qualcomm.robotcore.eventloop.opmode.Autonomous;
import com.qualcomm.robotcore.eventloop.opmode.Disabled;
import org.firstinspires.ftc.teamcode.CDDriveChassisAuton;

@Autonomous(name="CDAutonBaseTest", group="Linear Opmode")
@Disabled
public class CDAutonBaseTest extends CDAutonBase {
//    @Override
//    public void initTokenWeDoNotSee() {
//        duckWeDoNotSee = 1;
//    }

    @Override
    public void executeAuton() {
        myChassis.encoderDriveStrafe(org.firstinspires.ftc.teamcode.CDDriveChassisAuton.DRIVE_SPEED, 2, 5);
        myChassis.encoderDriveStraight(CDDriveChassisAuton.DRIVE_SPEED, 2, 10.0);
        myChassis.encoderDriveStrafe(org.firstinspires.ftc.teamcode.CDDriveChassisAuton.DRIVE_SPEED, -2, 5);
        myChassis.encoderDriveStraight(CDDriveChassisAuton.DRIVE_SPEED, 2, -10.0);
    }

}

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/CDAutonBaseTest.kt
package org.firstinspires.ftc.teamcode

import com.qualcomm.robotcore.eventloop.opmode.Autonomous
import com.qualcomm.robotcore.eventloop.opmode.Disabled

@Autonomous(name = "CDAutonBaseTest", group = "Linear Opmode")
class CDAutonBaseTest : CDAutonBase() {
    override fun initTokenWeDoNotSee() {
        duckWeDoNotSee = 1
    }

    override fun executeAuton() {
        myChassis.encoderDriveStrafe(CDAutonDriveChassis.DRIVE_SPEED, 2.0, 5.0)
        myChassis.encoderDriveStraight(CDAutonDriveChassis.DRIVE_SPEED, 2.0, 10.0)
        myChassis.encoderDriveStrafe(CDAutonDriveChassis.DRIVE_SPEED, -2.0, 5.0)
        myChassis.encoderDriveStraight(CDAutonDriveChassis.DRIVE_SPEED, 2.0, -10.0)
    }
}